keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -106,62 +306,62 @@ type RollingUpdateStatefulSetStrategy struct {
 
         1. PMP=Parallel will now apply to RollingUpdate. This will choose behavior described in 2 above.
-           This means always maxUnavailable number of Pods are terminating at the same time except in 
+           This means always maxUnavailable number of Pods are terminating at the same time except in
            the last case and no ordering guarantees are provided.
-        2. PMP=OrderedReady with maxUnavailable can choose one of behavior 1 or 3. 
+        2. PMP=OrderedReady with maxUnavailable can choose one of behavior 1 or 3.
 
-NOTE: The goal is faster updates of an application. In some cases , people would need both ordering 
-and faster updates. In other cases they just need faster updates and they dont care about ordering as 
+NOTE: The goal is faster updates of an application. In some cases , people would need both ordering
+and faster updates. In other cases they just need faster updates and they dont care about ordering as
 long as they get identity.
 
-Choice 1 is simpler to reason about. It does not always have maxUnavailable number of Pods in 
-Terminating state. It does not guarantee ordering within the batch of maxUnavailable Pods. The maximum 
+Choice 1 is simpler to reason about. It does not always have maxUnavailable number of Pods in
+Terminating state. It does not guarantee ordering within the batch of maxUnavailable Pods. The maximum
 difference in the ordinals which are Terminating out of Order, cannot be more than maxUnavailable.
 
-Choice 2 always offers maxUnavailable number of Pods in Terminating state. This can sometime lead to 
+Choice 2 always offers maxUnavailable number of Pods in Terminating state. This can sometime lead to
 pods terminating out of order. This will always lead to the fastest rollouts. The maximum difference in the ordinals which are Terminating out of Order, can be more than maxUnavailable.
 
-Choice 3 always guarantees than no two pods are ever Terminating out of order. It sometimes does that, 
-at the cost of not being able to Terminate maxUnavailable pods. The implementationg for this might be 
+Choice 3 always guarantees than no two pods are ever Terminating out of order. It sometimes does that,
+at the cost of not being able to Terminate maxUnavailable pods. The implementationg for this might be
 complicated.
 
-Choice 4 provides a choice to the users and hence takes the guessing out of the picture on what they 
+Choice 4 provides a choice to the users and hence takes the guessing out of the picture on what they
 will expect. Implementing Choice 4 using PMP would be the easiest.
 
 #### Implementation
 
 The alpha release we are going with Choice 4 with support for both PMP=Parallel and PMP=OrderedReady.
 For PMP=Parallel, we will use Choice 2
-For PMP=OrderedReady, we will use Choice 3 to ensure we can support ordering guarantees while also 
+For PMP=OrderedReady, we will use Choice 3 to ensure we can support ordering guarantees while also
```

Comment:
It seems to me we are using Choice one with `PMP=OrderedReady` in our implementation. 

this part does not happen today as described in choice 3

> At this time both 2 and 3 are terminating.

Can you please double check the choices and see if they are up to date?

Reply:
+1

Also please update if we're continuing with the choice for Beta or are we changing something?

Reply:
I went over it and fixed the text. Also I prefer the simpler route here and not change expected behavior. If behavior would change I would keep this in alpha 1 or two releases before coming back to promotion again.

I added the fix of the newly found bug in the implementation description

Reply:
I do not see big drawbacks using the simpler choice 1 instead of choice 3, but it might be a good idea to discuss this with a sig.